__tests__/_mockData.js

```diff
@@ -82,12 +82,27 @@ module.exports = {
       "date": 1605292482741,
       "summary": "Eligendi quae nihil nihil qui dolor ut.",
       "body": "Ullam nihil consequatur in magnam. Maiores vero quisquam facere dolorum eos est veritatis in id. Necessitatibus quos aut molestiae quia. Quae unde beatae.",
-      "recommend": "true",
-      "reported": "false",
+      "recommend": true,
+      "reported": true,
       "reviewer_name": "Curtis_King46",
       "reviewer_email": "Dolly.Muller75@hotmail.com",
-      "response": "null",
+      "response": null,
       "helpfulness": 12
+    },
+    {
+      "_id": "60fad51d93cd1365f0ae7cad",
```

Comment:
alpha-numeric identifier? did you implement some indexing or user id system?

Reply:
This is just what mongo does. When I imported it, I probably should have made it the id's that are provided, but I didn't and here we are hahaha